### ISSUE TITLE
Autocompletion doesn't work on `get_node(".")`

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#30812]
Godot 3.1.1

Provided this piece of code:
```gdscript
extends Spatial

# Called when the node enters the scene tree for the first time.
func _ready():
	tran # works
        self.tran # works
	get_node(".").tran #gives nothing
```

Note: I tested this with the script present in the current scene.

![image](https://user-images.githubusercontent.com/1311555/61876637-43d09e00-aee5-11e9-9c09-3934fade0620.png)

**Comment by yamgent** [comment]
Bug seems to be fixed in master branch by commit f9d9530.

**Comment by akien-mga** [comment]
Fixed by #28609.
